pkg/visor/init.go

```diff
@@ -344,6 +345,59 @@ func initDmsgCtrl(ctx context.Context, v *Visor, _ *logging.Logger) error {
 	return nil
 }
 
+func initDmsgHTTPLogServer(ctx context.Context, v *Visor, log *logging.Logger) error {
+	dmsgC := v.dmsgC
+	if dmsgC == nil {
+		return nil
+	}
+	var dmsgTimeout = v.conf.LogRotationInterval
+	logger := v.MasterLogger().PackageLogger("dmsghttp_logserver").WithField("timeout", dmsgTimeout)
+
+	lis, err := dmsgC.Listen(uint16(uint(80)))
+	if err != nil {
+		return err
+	}
+	go func() {
+		<-ctx.Done()
+		if err := lis.Close(); err != nil {
+			logger.WithError(err).Error()
+		}
+	}()
+	srv := &http.Server{
+		ReadHeaderTimeout: 5 * time.Second,
+		ReadTimeout:       5 * time.Second,
+		WriteTimeout:      time.Duration(dmsgTimeout),
+		Handler:           http.FileServer(http.Dir(v.conf.LocalPath)),
```

Comment:
Should this serve the whole `local` dir or just a subfolder? Is there data inside the local dir that the server should not expose?

Reply:
This is just temporary. We can update this part once the relevant PR's are worked on which will populate the required folder.